src/pages/learn/awesome_lucky/index_page.cr

```diff
@@ -0,0 +1,31 @@
+class Learn::AwesomeLucky::IndexPage < PageLayout
+  def hero_content
+    render_hero_content
+  end
+
+  def content
+    # https://github.com/andrewmcodes/awesome-lucky
+    div class: "py-4 container text-center" do
+      text "Coming soon..."
```

Comment:
These pages display a "Coming soon..." for now which will allow us to do these in separate PRs